tensorflow_addons/layers/optical_flow.py

```diff
@@ -141,6 +141,33 @@ def _correlation_cost_grad(op, grad_output):
 
 @keras_utils.register_keras_custom_object
 class CorrelationCost(tf.keras.layers.Layer):
+    """
+    This layer implements the correlation operation from 
+    FlowNet Learning Optical Flow with Convolutional Networks (Fischer et al.) r
+
+    Following are the parameters it takes:
+        input_a: A `Tensor` of the format specified by `data_format`. 
```

Comment:
Hi @PyExtreme! Thanks for the contribution! Would you mind adapting this doc string parameters to match the `__init__` args that are used to construct this layer. For example you'll notice input_a and input_b are passed in on the Layers `__call__` method. There needs to be some slight changes since this doc string is for the Layer's instantiation.